python_modules/dagster/dagster_tests/core_tests/log_tests/test_logging.py

```diff
@@ -256,3 +272,90 @@ def logged_solid(context):
 
     assert re.search("test OUTPUT debug logging from logged_solid.", captured.err, re.MULTILINE)
     assert re.search("test INPUT debug logging from logged_solid.", captured.err, re.MULTILINE)
+
+
+def test_conf_file_logging():
+    with tempfile.TemporaryDirectory() as tempdir:
+        with open(file_relative_path(__file__, "dagster.yaml"), "r") as template_fd:
+            with open(os.path.join(tempdir, "dagster.yaml"), "w") as target_fd:
+                template = template_fd.read().format()
+                target_fd.write(template)
+
+        instance = DagsterInstance.from_config(tempdir)
+
+        @solid
+        def log_solid(context):
+            context.log.info("Hello world!")
+
+        @pipeline
+        def log_pipeline():
+            log_solid()
+
+        # Python 3.4+
```

Comment:
The `redirect_stdout` functionality might not work for python versions < 3.4. Will this be a problem?